doc/make.bat

```diff
@@ -37,5 +38,11 @@ goto end
 :help
 %SPHINXBUILD% -M help %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
 
+:pdf

```

Comment:
We should try if the PDF build in Windows works before merging.. just to check

Reply:
@RobPasMue I checked and in my system it was working.